spec/integration/api/users_controller_spec.rb

```diff
@@ -6,7 +6,8 @@
   include PeopleHelper
 
   let(:full_scopes) {
-    %w[openid public:read public:modify private:read private:modify contacts:read profile profile:modify]
+    %w[openid public:read public:modify private:read private:modify
+      contacts:read contacts:modify profile profile:modify]
```

Comment:
Layout/AlignArray: Align the elements of an array literal if they span more than one line.